src/utils/getSentencesFromText.js

```diff
@@ -0,0 +1,19 @@
+export function getSentencesFromText(text, amount = 0) {
+
+  if (text === '' || null || undefined) return "";
```

Comment:
invalid syntax
right one is `(text === '' || text === null || text === undefined)`
i advise to use `if (!text) return ''` here bc similar effect

Reply:
Done